packages/react-native-codegen/src/parsers/parser.js

```diff
@@ -10,11 +10,16 @@
 
 'use strict';
 
+import type {ParserType} from './errors';
+
 /**
  * This is the main interface for Parsers of various languages.
  * It exposes all the methods that contain language-specific logic.
  */
 export interface Parser {
+  getMaybeEnumMemberType(maybeEnumDeclaration: $FlowFixMe): string;
+  isEnumDeclaration(maybeEnumDeclaration: $FlowFixMe): boolean;
+  language(): ParserType;
```

Comment:
Can we document those methods as this is the entrypoint of our API?

Reply:
Sure @cortinico, please let me know if these comments that I added are fine or if we should replace them with something else